frontend/apps/core-app/src/screens/RecipientRegistrationScreen/__tests__/RecipientRegistrationScreen.container.test.tsx

```diff
@@ -0,0 +1,251 @@
+      () => Promise.resolve(),
+      { data: forms, loading: false, error: null },
+    ]);
+
+  render(<RecipientRegistrationScreenContainer />);
+
+  expect(hooks.useAPICall).toHaveBeenCalledTimes(2);
+  expect(hooks.useAPICall).toHaveBeenCalledWith(
+    formsClient.Form.getAncestors,
+    [configuration.recipient.registrationForm.formId],
+    true,
+  );
+});
+
+describe('mode', () => {
+  describe('edit', () => {
+    it('should render', async () => {
+      jest
+        .spyOn(hooks, 'useAPICall')
+        .mockImplementation(() => [
+          () => Promise.resolve(),
+          { data: forms, loading: false, error: null },
+        ]);
+
+      const { getByText, getByTestId } = render(
+        <RecipientRegistrationScreenContainer />,
+      );
+
+      await waitFor(() => {
+        expect(getByText('edit')).toBeTruthy();
+        expect(
+          JSON.parse(getByTestId('mock-data').children[0].toString()),
+        ).toEqual(data);
+        expect(getByText('no-error')).toBeTruthy();
+        expect(getByText('loading - false')).toBeTruthy();
+      });
+    });
+
+    it('should call onSubmit', async () => {
+      jest
+        .spyOn(hooks, 'useAPICall')
+        .mockImplementation(() => [
+          () => Promise.resolve(),
+          { data: forms, loading: false, error: null },
+        ]);
+
+      const { getByText, getByTestId } = render(
+        <RecipientRegistrationScreenContainer />,
+      );
+
+      fireEvent.press(getByText('Submit'));
+
+      expect(getByText('review')).toBeTruthy();
+
+      await waitFor(() =>
+        expect(
+          JSON.parse(getByTestId('mock-data').children[0].toString()),
+        ).toEqual(mockSubmitData),
+      );
+    });
+
+    it('should call onCancel', () => {
+      jest
+        .spyOn(hooks, 'useAPICall')
+        .mockImplementation(() => [
+          () => Promise.resolve(),
+          { data: forms, loading: false, error: null },
+        ]);
+
+      const { getByText } = render(<RecipientRegistrationScreenContainer />);
+
+      fireEvent.press(getByText('Cancel'));
+
+      expect(mockNavigate).toHaveBeenCalledWith({
+        key: linkingConfig.config.screens.Recipients,
+      });
+    });
+  });
+});
+
+describe.only('review', () => {
+  it('should render', async () => {
+    jest
+      .spyOn(hooks, 'useAPICall')
+      .mockImplementation(() => [
+        () => Promise.resolve(),
+        { data: forms, loading: false, error: null },
+      ]);
+
+    const { getByText, getByTestId } = render(
+      <RecipientRegistrationScreenContainer />,
+    );
+
+    fireEvent.press(getByText('Submit'));
+
+    await waitFor(() => {
+      expect(getByText('review')).toBeTruthy();
+      expect(
+        JSON.parse(getByTestId('mock-data').children[0].toString()),
+      ).toEqual(mockSubmitData);
+      expect(getByText('no-error')).toBeTruthy();
+      expect(getByText('loading - false')).toBeTruthy();
+    });
+  });
+
+  // it('should call onSubmit', () => {
```

Comment:
you could just skip it? that way it would still show up in the test results and not be forgotten